lib/librato/rails/subscribers/job.rb

```diff
@@ -0,0 +1,15 @@
+module Librato
+  module Rails
+    module Subscribers
+      hooks = %w{enqueue_at enqueue perform_start perform}
+
+      hooks.each do |hook|
+        ActiveSupport::Notifications.subscribe "#{hook}.active_job" do |*args|
+          collector.group 'rails.job' do |c|
+            c.increment hook
```

Comment:
No, the only keys provided are `:adapter` and `:job`, so we're just incrementing: http://edgeguides.rubyonrails.org/active_support_instrumentation.html#active-job

Having said that, Active Job does provide `before_perform` and `after_perform` callbacks: http://edgeguides.rubyonrails.org/active_job_basics.html#callbacks

This could easily allow you to determine job duration. Planned on eventually opening a PR: https://github.com/rails/rails/blob/master/activejob/lib/active_job/logging.rb